post.md

```diff
@@ -27,17 +27,17 @@ I remember feeling slightly patronised by that.
 Why can't I, as an _application author_, become master of my own destiny and make use of this powerful feature?
 Is it too complex for mere mortals to comprehend?
 
-What the react authors don't know is that in the intervening years I have become a
+What the React authors don't know is that in the intervening years I have become a
 [data fetching framework author](https://www.npmjs.com/package/@jaybeeuu/recoilless).
 So I think it's time to exercise my prerogatives.
 
-In this post I'm going to try to fill in the blanks in the `Suspense` documentation. How do you make components suspend & handle promises in a react application to build a fantastic UX?
+In this post I'm going to try to fill in the blanks in the `Suspense` documentation. How do you make components suspend & handle promises in a React application to build a fantastic UX?
 
 Let's see if I can sneak you in with my backstage pass, shall we?
```

Comment:
I don't quite get it, do you _have_ to be a data fetching framework author before you are allowed to glimpse at the super secret documentation?